apps/speed.c

```diff
@@ -2011,14 +2011,17 @@ int speed_main(int argc, char **argv)
         case OPT_ASYNCJOBS:
 #ifndef OPENSSL_NO_ASYNC
             async_jobs = opt_int_arg();
+            if (async_jobs > 99999) {
```

Comment:
In my opinion, the `ASYNC_is_capable()` check should be performed first, followed by the `async_jobs` count check. Typically, we verify the supportability of a feature before checking other parameters or details.

Reply:
Nope, this would break the test recipe.